android-app/app/src/androidTest/java/de/dhbw/ka/se/fibo/ui/adding/AddingFragmentTest.java

```diff
@@ -274,4 +287,56 @@ public void TestInvalidAddressFieldInput() {
         onView(withId(R.id.address_text_layout))
                 .check(matches(hasTextInputLayoutErrorText("")));
     }
+
+    @Test
+    public void testFutureDateNotPossible() {
+        DateTimeFormatter shortFormat = DateTimeFormatter.ofPattern("dd.MM.yy");
+        LocalDateTime now = LocalDateTime.now();
+        String today = shortFormat.format(now);
```

Comment:
```suggestion
        DateTimeFormatter shortFormat = DateTimeFormatter.ofPattern("dd.MM.yy");
        String today = shortFormat.format(LocalDateTime.now());
        String tomorrow = shortFormat.format(LocalDateTime.now().plusDays(1));
```

How about something like this? I think later on it will be a bit clearer that the test is about the **next** day and not just some random day in the future